driver/sql/postgres/eventstore.go

```diff
@@ -206,6 +207,7 @@ func (e *EventStore) AppendToWithExecer(ctx context.Context, conn driverSQL.Exec
 
 	result, err := conn.ExecContext(
 		ctx,
+		/* #nosec */
```

Comment:
What is this? :eyes:

Reply:
By using this command we tell gosec to ignore the possible SQL injection warning. (for more information see https://github.com/securego/gosec#configuration)